templates/sshd_config.erb

```diff
@@ -238,7 +238,7 @@ XAuthLocation <%= @sshd_config_xauth_location_real %>
 
 <% end -%>
 # override default of no subsystems
-Subsystem sftp <%= @sshd_config_subsystem_sftp_real %>
+Subsystem sftp <%= @sshd_config_subsystem_sftp_real %> <%= @sshd_config_subsystem_sftp_options %>
```

Comment:
the trailing whitespace is causing an issue with the spec tests. You can enclose it and the part you added in an if statement as shown above.